semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +120,48 @@ p strong {
   margin-left: 10px;
 }
 
+/*--------------------------------
+   Application home & create
+---------------------------------*/
+#application-area {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.company-title {
+  letter-spacing: @letterSpacingRegular;
+  line-height: @lineHeightBig;
+}
+
+// Back to application list label
+#application-area .label {
+  align-self: flex-start;
+  min-width: 132px;
+  max-width: 500px;
+  height: 25px;
+  background-color: transparent;
+}
+
+/*--------------------------------
+      General used classes
+---------------------------------*/
+
+// In use for ApplicationCreate and ApplicationHome
+#application-content {
```

Comment:
IDs should only appear ONCE, they're meant to be unique identifiers. If it's going in multiple places, then just make it a `.class`.

Reply:
Oh, All good, thanks for pointing that out

Reply:
If I change this to be a class then it is not finding other styles (defined used this class included). Can you give me a hand with that please?